### PR TITLE
syntax highlighting in action logger

### PR DESCRIPTION
**Pull request by ritz078**
screen shot 2016-04-19 at 12 27 56 am
screen shot 2016-04-19 at 12 28 12 am

### CODE REVIEW

File: src/client/ui/__tests__/foldable.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ describe('<Foldable />', function () {
         args: 'things',
       };
 
-      const fullString = '{\n  "name": "test action",\n  "args": "things"\n}';
+      const fullString = '{\n  name: "test action",\n  args: "things"\n}';
```
Comment (arunoda) [inline src/client/ui/__tests__/foldable.js]: Can't we write an additional test case targetted for syntax highlighting.

File: src/client/ui/highlight.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+export default function highlight(data) {
```
Comment (arunoda) [inline src/client/ui/highlight.js]: Is there a way we can use a NPM module for this?

**Comment by arunoda** [comment]
Overall, this looks cool. 
Once we are okay with suggestions I made, we could take this in.

File: src/client/ui/__tests__/foldable.js (hunk 1 above)
Comment (ritz078) [inline src/client/ui/__tests__/foldable.js]: sure

File: src/client/ui/highlight.js (hunk 2 above)
Comment (ritz078) [inline src/client/ui/highlight.js]: couldn't find a module that did only this. Most of them were overkill as they had separate CSS and were made to parse different languages.

Comment (arunoda) [inline src/client/ui/highlight.js]: Okay. Then that's fine.
Try to make some comments on why we are using it and so on.

**Comment by ritz078** [comment]
@arunoda added tests and comment

**Comment by arunoda** [comment]
Okay. Great.

**Comment by arunoda** [comment]
Published as `v1.14.0`

**Pull request merged**